psycop/common/model_training_v2/test_pipeline.py

```diff
@@ -33,38 +42,60 @@
 from psycop.common.test_utils.str_to_df import str_to_pl_df
 
 
+@BaselineRegistry.baseline_data.register("minimal_test_data")
+class MinimalTestData(BaselineDataLoader):
+    def __init__(self) -> None:
+        pass
+
+    def load(self) -> LazyFrame:
+        data = str_to_pl_df(
+            """pred_time_uuid,pred_1,outcome,pred_age
+                                        1,1,1,1
+                                        2,1,1,99
+                                        3,1,1,99
+                                        4,0,0,99
+                                        5,0,0,99
+                                        6,0,0,99
+                                        """,
+        ).lazy()
+
+        return data
+
+
 def test_v2_train_model_pipeline(tmpdir: Path):
-    training_data = str_to_pl_df(
-        """pred_time_uuid,pred_1,outcome,pred_age
-                                     1,1,1,1
-                                     2,1,1,99
-                                     3,1,1,99
-                                     4,0,0,99
-                                     5,0,0,99
-                                     6,0,0,99
-                                     """,
-    )
     logger = TerminalLogger()
     schema = BaselineSchema(
-        experiment_path=tmpdir,
+        project_info=ProjectInfo(experiment_path=tmpdir),
         logger=logger,
         trainer=SplitTrainer(
-            training_data=training_data,
+            training_data=MinimalTestData(),
```

Comment:
@HLasse Confection instantiates all the elements at resolve-time, so having a class made more sense. Perhaps we could make a generic loader which just takes a .parquet path as an argument?